ntbs-service/Services/AlertService.cs

```diff
@@ -124,6 +125,20 @@ public async Task DismissAlertAsync(int alertId, string userId)
             return true;
         }
 
+        public async Task AddUniquePotentialDuplicateAlertAsync(DataQualityPotentialDuplicateAlert alert)
+        {
+            if (alert.NotificationId.HasValue)
+            {
+                var matchingAlert = await _alertRepository.GetDuplicateAlertByNotificationIdAndDuplicateId(alert.NotificationId.Value, alert.DuplicateId);
+                if (matchingAlert != null)
+                {
+                    return;
+                }

```

Comment:
I quite like these being the short-hand version, what do you think?
```suggestion
                if (matchingAlert != null) return;
```